pkg/operator/storageclasscontroller/vmware.go

```diff
@@ -326,6 +331,7 @@ func (c *StorageClassController) getCredentials(cfg *vsphere.VSphereConfig) (str
 }
 
 func notFoundError(err error) bool {
+	// httpError := err.(*http.Response)
```

Comment:
can you please remove the commented line?

Reply:
fixed